cdc/owner.go

```diff
@@ -367,13 +367,17 @@ func (o *Owner) newChangeFeed(
 		sinkTableInfo[j-1] = new(model.SimpleTableInfo)
 		sinkTableInfo[j-1].TableID = tid
 		sinkTableInfo[j-1].ColumnInfo = make([]*model.ColumnInfo, len(tblInfo.Cols()))
+		// forget to assign value to schema and table, or file log.meta about this content are all empty
+		sinkTableInfo[j-1].Schema = table.Schema
```

Comment:
@3pointer , It seems that this point has been fixed see #1672 , should this PR need closed?

Reply:
> @3pointer , It seems that this point has been fixed see #1672 , should this PR need closed?

@3pointer could u help confirm, thanks

Reply:
Yes, changed here can be reverted.